.github/workflows/containers/github-action-ci/Dockerfile

```diff
@@ -66,6 +66,11 @@ RUN apt-get update && \
     file \
     tzdata
 
+# Install sccache as it is needed by most of the project test workflows and
+# cannot be installed by the ccache action when executing as a non-root user.
+RUN curl -L 'https://github.com/mozilla/sccache/releases/download/v0.7.6/sccache-v0.7.6-x86_64-unknown-linux-musl.tar.gz' | tar xzf - -O --wildcards '*/sccache' > '/usr/local/bin/sccache' && \
+    chmod +x /usr/local/bin/sccache
```

Comment:
Can we use the package manager to install sccache?

Reply:
It's not packaged by default for most distros. That's what I would have preferred too.

Reply:
Ok, looks like it wasn't added to Ubuntu until 24.04.  Can you add a TODO reminding us to install using apt-get whenever we bump the base image up to 24.04.  Also, can you do a cheksum check on the download like they do in the action we've been using:  https://github.com/hendrikmuhs/ccache-action/blob/main/src/restore.ts#L147
That will be safer and will make this change essentially a no-op.

Reply:
Added a todo and added the checksum.